jetty-alpn/pom.xml

```diff
@@ -10,6 +10,10 @@
   <artifactId>jetty-alpn-parent</artifactId>
   <packaging>pom</packaging>
   <name>Jetty :: ALPN :: Parent</name>
+  <properties>
+    <conscrypt.version>1.0.0.RC11</conscrypt.version>
```

Comment:
I think it is better that all versions reside in the main POM. In this way, we only need to look at the main POM when we want to update versions for dependencies. I would move this property in the main POM.